examples/chip-tool/BUILD.gn

```diff
@@ -25,6 +25,10 @@ if (config_use_interactive_mode) {
 
 assert(chip_build_tools)
 
+declare_args() {
+  libs = []
```

Comment:
libs seems quite a generic argument name. Can we be more specific? 

Please add comments explaining purpose and usage.

Reply:
Why does chip-tool need this? I see darwin-framework-tool uses it only.

Reply:
> libs seems quite a generic argument name. Can we be more specific?

This is a built-in gn variable.  See https://gn.googlesource.com/gn/+/main/docs/reference.md#var_libs

Reply:
By "uses" I mean to justify use cases in comments. This PR asks to add something that we need to then maintain and because nobody uses it, we won't have CI for this. Maybe we should have some CI (e.g. some malloc library for testing? I have no idea)...

I am pushing back on adding something that is very broad (can add any library here) that has no tests and no dscription of practical use. I am use it has a practical use in mind, but nothing documented and nothing in CI/tested is concerning.